pkg/config/cli_config.go

```diff
@@ -55,3 +60,11 @@ func (cfg *CLIConfig) SetDefault() {
 		cfg.Version = "1"
 	}
 }
+
+// Default to server's doman if JWKSEndpoint is not specified.
+func (cfg *CLIConfig) GetJWKSEndpoint() string {
+	if cfg.JWKSEndpoint == "" && cfg.Server != "" {
+		return fmt.Sprintf("https://%s:8080/.well-known/jwks", strings.Split(cfg.Server, ":")[0])
+	}
+	return cfg.JWKSEndpoint
+}
```

Comment:
Can we make this part of `SetDefault()` instead of a separate func?

Reply:
Agreed. Also naming functions Get/Set is generally frowned upon outside of auto-generated code.